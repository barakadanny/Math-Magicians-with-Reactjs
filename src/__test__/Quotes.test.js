import React from 'react';
import renderer from 'react-test-renderer';
import Quotes from '../components/Quote';
import '@testing-library/jest-dom';

describe('Test Quotes pag', () => {
  it('testing Quotes page UI rendering', () => {
    const tree = renderer.create(<Quotes />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
