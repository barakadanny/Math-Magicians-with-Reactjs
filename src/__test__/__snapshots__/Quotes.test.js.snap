// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Quotes pag testing Quotes page UI rendering 1`] = `
<div
  className="container quote"
>
  <p>
    Mathematics is not about numbers, equations, computations,
     
    <br />
     
    or algorithm; it is about understanding. -William Paul Thurston.
  </p>
</div>
`;
